companion.txt

```diff
@@ -27,3 +27,11 @@ sympy>=1.9
 # Needed for KDE trigger statistics
 git+https://github.com/mennthor/awkde.git@master
 scikit-learn
+
+# Needed for `BBHx` package to work with PyCBC
+gcc_linux-64
+gxx_linux-64
+gsl
+lapack=3.6.1
```

Comment:
@WuShichao These aren't python packages. Remember you need to put this in tox.ini. I also doubt you need the gcc / gxx parts, but perhaps you might. Add one at a time so we don't do anything unnecessarily. 

Second, the error you posted is because it can't even process the line, this isn't quite the syntax for setting a version requirement even it were a python package. See the requirements.txt file for how versions are done for future reference.